frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -240,6 +241,7 @@ export default {
           type,
           propagate: false,
           index,
+          ...DATA_TYPES_COMPONENTS[type],
```

Comment:
Sorry, I didn't see this line was added in the commit:
```suggestion
          ...DATA_TYPES_COMPONENTS[type],
```